applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -44,12 +46,38 @@ static struct net_buf_pool *iso_tx_pools[] = { LISTIFY(CONFIG_BT_ASCS_ASE_SRC_CO
 /* clang-format on */
 #endif /* CONFIG_STREAM_BIDIRECTIONAL */
 
+static struct bt_csis *csis;
 static struct bt_le_ext_adv *adv_ext;
 /* Advertising data for peer connection */
+static uint8_t csis_rsi[BT_CSIS_RSI_SIZE];
 static const struct bt_data ad_peer[] = {
 	BT_DATA_BYTES(BT_DATA_FLAGS, (BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR)),
 	BT_DATA_BYTES(BT_DATA_UUID16_ALL, BT_UUID_16_ENCODE(BT_UUID_ASCS_VAL)),
-	BT_DATA_BYTES(BT_DATA_UUID16_ALL, BT_UUID_16_ENCODE(BT_UUID_PACS_VAL))
+	BT_DATA_BYTES(BT_DATA_UUID16_ALL, BT_UUID_16_ENCODE(BT_UUID_PACS_VAL)),
+	BT_CSIS_DATA_RSI(csis_rsi)
+};
+
+static void csis_lock_changed_cb(struct bt_conn *conn, struct bt_csis *csis, bool locked)
+{
+	LOG_INF("Client %p %s the lock", (void *) conn, locked ? "locked" : "released");
+}
+
+static uint8_t sirk_read_req_cb(struct bt_conn *conn, struct bt_csis *csis)
+{
+	return BT_CSIS_READ_SIRK_REQ_RSP_ACCEPT_ENC;
+}
+
+static struct bt_csis_cb csis_cbs = {
+	.lock_changed = csis_lock_changed_cb,
+	.sirk_read_req = sirk_read_req_cb,
+};
+struct bt_csis_register_param csis_param = {
+	.set_size = 2,
+	.lockable = true,
+	/* Using the CSIS test sample SIRK */
+	.set_sirk = { 0xcd, 0xcc, 0x72, 0xdd, 0x86, 0x8c, 0xcd, 0xce, 0x22, 0xfd, 0xa1, 0x21, 0x09,
```

Comment:
Must have a todo to change this, or this should be generated based on some seed. If this is set the same on all audio kits, what happens if there are four headsets in one place and suddenly a gateweay connects to one of them? I guess it will only connect to one more, since set_size is 2, but will it be randomly selected?